tools/logger/convert.c

```diff
@@ -950,13 +950,16 @@ static int logger_read(void)
 	return ret;
 }
 
-/* fw verification */
-static int verify_fw_ver(void)
+/** Compare the dictionary checksum in the firmware image (through
+ * /sys/kernel/debug/sof/fw_version) with the checksum in the .ldc file.
+ * @return 0 when the checksums match
+ */
+static int verify_ldc_checksum(const uint32_t ldc_sum)
 {
 	struct sof_ipc_fw_version ver;
 	int count;
 
-	if (!global_config->version_fd)
+	if (!global_config->version_fw) // -n option
```

Comment:
C style

Reply:
Wait, I thought #5046 meant this is allowed now?

Reply:
Fine in the SPDX, but my pattern matching needs C style in the code.

Reply:
Done.